lemmy_utils/src/request.rs

```diff
@@ -88,7 +88,7 @@ pub(crate) async fn fetch_pictrs(
   is_image_content_type(client, image_url).await?;
 
   let fetch_url = format!(
-    "http://pictrs:8080/image/download?url={}",
+    "http://localhost:1235/image/download?url={}",
```

Comment:
I assume that this should be pointed to `Settings.pictrs_url`, but wanted to make sure before switching it.

Reply:
You're right, looks like we forgot to adjust that.